crates/viewer/re_data_ui/src/video.rs

```diff
@@ -113,7 +113,9 @@ fn video_data_ui(ui: &mut egui::Ui, ui_layout: UiLayout, video_data: &VideoData)
                     .value_text(video_data.gops.len().to_string()),
             )
             .on_hover_text("The total number of Group of Pictures (GOPs) in the video.");
-            samples_statistics_ui(ui, &video_data.samples_statistics);
+            ui.list_item_flat_noninteractive(
+                PropertyContent::new("All PTS equal DTS").value_bool(video_data.samples_statistics.dts_always_equal_pts)
```

Comment:
It would be nice to destruct `samples_statistics` here so that if we add new fields to it, we remember to also add a UI for it